chains/evm/contracts/pools/USDC/interfaces/IMessageTransmitter.sol

```diff
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Added a bunch of missing licenses because this PR hit some warning-limit. This limit is also fixed in foundry.toml but for completeness these fixes have also been included